### DISCUSSION TITLE
Not an issue, just a thank you!

### DISCUSSION THREAD

**Discussion by bashbunni** [NeogitOrg/neogit#514] [General]
Just wanted to say to all of the contributors THANK YOU!

I love Neogit and thank you so much for your hard work in making this reliable and free to use 🎊  You're awesome!!

**Comment by ten3roberts** [comment]
Thank you too. It makes me glad to hear that people are finding this plugin useful; open source is a lovely thing 😊 

Let me know if there are any features you need to suit your workflow

**Comment by CKolkey** [comment]
Always a delight to hear! We have some big upgrades planned over the next few months, so be sure to check back 😇

**Comment by MaxKiv** [comment]
+1 on this `issue` 😄 
I recently discovered neogit and I am amazed by how good this is, Thanks everyone!
